internal/operator-controller/applier/helm.go

```diff
@@ -209,9 +213,83 @@ func (h *Helm) buildHelmChart(bundleFS fs.FS, ext *ocv1.ClusterExtension) (*char
 	if err != nil {
 		return nil, err
 	}
+	if features.OperatorControllerFeatureGate.Enabled(features.HelmChartSupport) {
+		chart := new(chart.Chart)
+		if IsBundleSourceHelmChart(bundleFS, chart) {
+			return enrichChart(chart, WithInstallNamespace(ext.Spec.Namespace))
+		}
+	}
```

Comment:
Could we add the unit tests for this code? WDYT?

Reply:
Sure. Working on tests right now.